client/containers/Pub/PubDocument/PubInlineSuggestedEdits.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useMemo } from 'react';
+
+import { buttons, FormattingBarSuggestedEdits } from 'components/FormattingBar';
+import { acceptSuggestedEdits } from 'client/components/Editor/plugins/suggestedEdits/resolve';
+
+import { usePubContext } from '../pubHooks';
+
+require('./pubInlineSuggestionMenu.scss');
+
+const shouldOpenBelowSelection = () => {
+	return ['Android', 'iPad', 'iPhone'].some((device) =>
+		navigator.userAgent.toLowerCase().includes(device.toLowerCase()),
+	);
+};
+
+const PubInlineSuggestedEdits = () => {
+	const { collabData, pubBodyState } = usePubContext();
+	const { editorChangeObject } = collabData;
+
+	const selection = collabData.editorChangeObject!.selection;
+
+	const shouldHide = useMemo(() => {
+		const selectionInSuggestionRange = (): boolean => {
+			if (!collabData.editorChangeObject || !collabData.editorChangeObject.view) return false;
+			const state = collabData.editorChangeObject.view.state;
+			return acceptSuggestedEdits(state);
+		};
+
+		const inRange = selectionInSuggestionRange();
+
+		return !inRange || !selection;
```

Comment:
```suggestion
		if (!collabData.editorChangeObject || !collabData.editorChangeObject.view) {
			return false;
		}
		const state = collabData.editorChangeObject.view.state;
		return !acceptSuggestedEdits(state); || !selection;
```

I don't think you need this function or the variables, since its only used here

Reply:
![image](https://github.com/pubpub/pubpub/assets/34730449/be910ae4-7f11-4b28-871c-beaaf201c6ea)


i get this error when applying that change, i tried doing this in the am and kept getting this err so maybe im doing something wrong?

Reply:
oh yeah me too. it's because that `return false;` should be a `return true;`: we do want to hide this menu when those objects aren't present. relatedly, i think it's a mistake to assert `collabData.editorChangeObject!` before this block, so here's a rewritten version that works for me:

```typescript
	const selection = collabData.editorChangeObject?.selection;

	const shouldHide = useMemo(() => {
		if (!collabData.editorChangeObject || !collabData.editorChangeObject.view || !selection) {
			return true;
		}
		const state = collabData.editorChangeObject.view.state;

		return !acceptSuggestedEdits(state);
	}, [collabData.editorChangeObject, selection]);
```

Reply:
ngl i was going crazy trying to figure out what was going on and stumbled into solving something you already did with acceptedEdits, i see that it really does always return true.  had to rebase again lol